### PR TITLE
Update the CI1, CI2 , & CI3 tasks to refer to 2017

### PR DESCRIPTION
**Pull request by namantw**
This commit changes all occurrences of 2016 to 2017 and the project
links with the new ones in `use_coala.md`, `use_coala_2.md` and
`use_coala_3.md`.

Closes https://github.com/coala/projects/issues/433

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by namantw** [comment]
Sorry that I messed up. I will update the commit message.

**Comment by namantw** [comment]
I have changed the commit message. Please check if it is okay.

**Comment by namantw** [comment]
I should have been more careful. It is updated now, please check @adi23arora

**Comment by namantw** [comment]
Done, please review @adi23arora

**Comment by namantw** [comment]
@adi23arora apparently my commit message had a trailing period. I have updated the commit 😃 . Please review.

> Shortlog
>
> Example:
> `setup.py: Change bears' entrypoint`
> Maximum of 50 characters.
> Keeping subject lines at this length ensures that they are readable, and explains the change in a concise way.
> Should describe the change - the action being done in the commit.
> 
> Should have a tag and a short description separated by a colon (:)
> 
> Tag
>  - The file or class or package being modified.
>  - Not mandatory.
>
> Short Description
>  - Starts with a capital letter.
>  - Written in imperative present tense (i.e. Add something, not Adding something or Added something).
>  - No trailing period.

**Comment by pareksha** [comment]
@namantw try to write commit messages in a commanding sense. Use words like `change` instead of `changes`

**Comment by namantw** [comment]
@pareksha I think there is no such 'rule' for commit body in the [commit guidelines](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html)
Quoting from [commit guidelines](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html)

> Example 2 (implemented feature):
> 
> Linter: Output command on debug
> 
> This massively helps debugging linters.
> 
> Closes https://github.com/coala/coala/issues/2060

Notice how they used 'helps' and not 'help'.

**Comment by pareksha** [comment]
@namantw, I am sorry I didn't mention it before. I am talking about the shortlog. Notice that in `Output command on debug`, they used 'Output' and not 'Outputs'.
Your commit body is totally correct. You can improve it by using a shortlog in a commanding sense.

**Comment by namantw** [comment]
@pareksha looks like you're confused regarding the 'shortlog' 

> Update the CI1, CI2 & CI3 tasks to refer to 2017 <---- This is the shortlog
> 
>  This commit changes all occurrences of 2016 to 2017 and the project
> links with the new ones in `use_coala.md`, `use_coala_2.md` and
> `use_coala_3.md`.
> 
> Closes #433 

'Update' is in "commanding sense". Hope that clears your doubt 😉

**Comment by pareksha** [comment]
Ah! Just forgot about that, my bad. Thanks for clearing it for me though, @namantw  :100:

**Comment by namantw** [comment]
@pareksha if you are satisfied with the PR, please do `ack https://github.com/coala/projects/pull/435/commits/de940c28608644a28ffc04ec4d23f65836c979cb` (without backticks) to approve 😀

File: _projects/use_coala.md [hunk 1]
```diff
@@ -74,13 +74,13 @@ can be used to help select a repository that meets those criteria.
...
-2. [`pushed:>2016-09-30`](https://github.com/search?utf8=%E2%9C%93&q=pushed%3A%3E2016-09-30&type=Repositories&ref=searchresults)
+2. [`pushed:>2017-09-30`](https://github.com/search?utf8=%E2%9C%93&q=pushed%3A%3E2017-09-30&type=Repositories&ref=searchresults)
 
-The [combined query](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A>2016-09-30&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
+The [combined query](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A>2017-09-30&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
 has more than 28,000 repository results, so there are many to choose from.
 
 An optional additional search term is
-[`forks:>100`](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A%3E2016-09-30+forks%3A%3E%3D100&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
+[`forks:>100`](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A%3E2017-09-30+forks%3A%3E%3D100&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
```
Comment (ishanSrt) [inline _projects/use_coala.md]: what is this link referring to actually? why have you given the date of 30th September 2017 thats so far ahead in the future and GitHub is still showing results. Shouldn't you be giving present date or something?

Comment (namantw) [reply _projects/use_coala.md]: The link is working fine, it refers to combined query = star:>99,  pushed :>2017-09-30 and forks:>100 as given in https://github.com/coala/projects/blob/master/_projects/use_coala.md

Comment (ishanSrt) [reply _projects/use_coala.md]: what does  pushed :>2017-09-30  mean?

Comment (namantw) [reply _projects/use_coala.md]: it means the repository has atleast one push after 30th September, 2017

Comment (ishanSrt) [reply _projects/use_coala.md]: shit got confused between 2017 and 18 🤪.  You are all good

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/435/commits/de940c28608644a28ffc04ec4d23f65836c979cb

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**